cmd/antrea-agent/options.go

```diff
@@ -171,6 +178,23 @@ func (o *Options) setDefaults() {
 	}
 }
 
+func (o *Options) validateTLSOptions() error {
+	_, err := cliflag.TLSVersion(o.config.TLSMinVersion)
+	if err != nil {
+		return fmt.Errorf("invalid TLSMinVersion: %v", err)
+	}
+	trimmedTLSCipherSuites := strings.ReplaceAll(o.config.TLSCipherSuites, " ", "")
```

Comment:
Line 186-192 is similar to cipher.GenerateCipherSuitesList(), the difference is new code has a check with `trimmedTLSCipherSuites != ""`, why not use this version to re-write GenerateCipherSuitesList?

Reply:
As explained above, the code in `pkg/util/cipher/` is duplicated with the code in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/cli/flag/ciphersuites_flag.go and https://github.com/kubernetes/kubernetes/blob/8206c9d458e321d7ad22ea9fc2e21a890790fc09/staging/src/k8s.io/apiserver/pkg/server/options/serving.go#L274-L286. The right way to set TLS configurations is setting the SecureServingOptions with raw strings instead of calculating the results and mutating APIServer directly. The difference is not only the check, but also the data type of the slice. We are passing `[]string` to SecureServingOptions, not `[]int16` to GenericAPIServer.

`pkg/util/cipher/` will be removed after all APIServers are changed to use upstream code, I don't want to touch antrea-controller and flow-aggregator code in this PR so keep them for now.